packages/apps/graph/src/graph/objects/Event.ts

```diff
@@ -0,0 +1,51 @@
+import { prismaClient } from '../../db/prismaClient';
+import { builder } from '../builder';
+
+export default builder.prismaNode('Event', {
+  id: { field: 'block_idx_requestkey' },
+  fields: (t) => ({
+    // database fields
+    id_gen: t.expose('id', { type: 'Int' }),
+    chainId: t.expose('chainid', { type: 'BigInt' }),
+    height: t.expose('height', { type: 'BigInt' }),
+    index: t.expose('idx', { type: 'BigInt' }),
+    module: t.exposeString('module'),
+    name: t.exposeString('name'),
+    qualName: t.exposeString('qualname'),
+    paramText: t.exposeString('paramtext'),
+    requestKey: t.exposeString('requestkey'),
+    parameters: t.field({
+      type: 'String',
+      resolve(parent) {
+        return JSON.stringify(parent.params);
+      },
+    }),
```

Comment:
From what I could see in the database, there do not seem to be any keys associated with the values. The structure is more similar to an array of values rather than a traditional JSON object with key-value pairs.
eg `["","k:f90ef46927f506c70b6a58fd322450a936311dc6ac91f4ec3d8ef949608dbf1f",1.1522615]`